src/app/pages/registration/registration-form-configuration/registration-form-configuration.service.ts

```diff
@@ -25,7 +25,8 @@ export interface RegistrationConfigType {
   cancelUrl?: string;
 }
 
-@Injectable({ providedIn: 'root' })
+@Injectable()
+// eslint-disable-next-line ish-custom-rules/project-structure
```

Comment:
you mean like `src/app/pages/registration/services`? I moved it here because it's very much a "utility" service that is only used by this page and only exists to make the component simpler and for seperation of concerns. Where would you rather place it?